File: solutions/studyplan/programmingskills/easy_709/Solution.java
package studyplan.programmingskills.easy_709;

import java.lang.reflect.Array;
import java.util.ArrayList;
import java.util.Arrays;
import java.util.Locale;
import java.util.Objects;

public class Solution {

    /*  My Solution
     *  Runtime: 0ms
     *  Memory: 42.77MB
     */
    public String toLowerCase(String s) {


        char[] chars = s.toCharArray();
        for (int i = 0; i < chars.length; i++) {
            char c = chars[i];
            if (c >= 'A' && c <= 'Z') {
                chars[i] = (char) (c + 32);
            }
        }
        return new String(chars);
    }

    /*  My Solution
     *  Runtime: 0ms
     *  Memory: 42.21MB
     */
    public String toLowerCase1(String s) {

        return s.toLowerCase();
    }
}

File: solutions/studyplan/programmingskills/easy_709/Solution.kt
package studyplan.programmingskills.easy_709

import java.sql.Array

/*  My Solution
   *  Runtime: 0ms
   *  Memory: 41.46MB
   */

fun toLowerCase(s: String): String {
    val chars = s.toCharArray()
    for (i in s.indices) {
        val c = s[i]

        if (c in 'A'..'Z') {
            chars[i] = (c.code or 32).toChar()
        }
    }
    return String(chars)
}
/*  My Solution
   *  Runtime: 0ms
   *  Memory: 41.18MB
   */

fun toLowerCase1(s: String): String {
    return s.lowercase()
}